src/constants/index.ts

```diff
@@ -29,6 +29,8 @@ export type RouteParams = {
 	filter?: 'member' | 'manager';
 };
 
+export type ZimbraHideInGalType = 'TRUE' | 'FALSE' | undefined;
```

Comment:
Move this type to the network file where the field is used. It should not be in constants file considering that it is a network type and it should be used in application logic. Also, consider creating a more generic type like BooleanString, like
```
type BooleanString = 'TRUE' | 'FALSE';
// or
type BooleanString = Uppercase<`${boolean}`>;
```